wpilibNewCommands/src/main/native/cpp/frc2/command/Command.cpp

```diff
@@ -175,21 +175,17 @@ void Command::InitSendable(wpi::SendableBuilder& builder) {
       });
   builder.AddBooleanProperty(
       ".isParented", [this] { return IsComposed(); }, nullptr);
-  builder.AddStringProperty(
-      "interruptBehavior",
-      [this] {
-        switch (GetInterruptionBehavior()) {
-          case Command::InterruptionBehavior::kCancelIncoming:
-            return "kCancelIncoming";
-          case Command::InterruptionBehavior::kCancelSelf:
-            return "kCancelSelf";
-          default:
-            return "Invalid";
-        }
-      },
-      nullptr);
-  builder.AddBooleanProperty(
-      "runsWhenDisabled", [this] { return RunsWhenDisabled(); }, nullptr);
+  std::string_view interruptBehaviorString{"Invalid"};
+  switch (GetInterruptionBehavior()) {
+    case Command::InterruptionBehavior::kCancelIncoming:
+      interruptBehaviorString = "kCancelIncoming";
+      break;
+    case Command::InterruptionBehavior::kCancelSelf:
+      interruptBehaviorString = "kCancelSelf";
+      break;
+  }
+  builder.PublishConstString("interruptBehavior", interruptBehaviorString);
+  builder.PublishConstBoolean("runsWhenDisabled", RunsWhenDisabled());
```

Comment:
C++ isn't sending the requirements in initSendable

Reply:
#5836 will be needed for that.